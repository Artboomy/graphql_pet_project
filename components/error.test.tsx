import * as React from 'react';
import renderer from 'react-test-renderer';
import ErrorView from './error';

test('Renders passed message', () => {
    const tree = renderer
        .create(<ErrorView errorMessage={'Test error message'} />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});

test('Renders default message', () => {
    const tree = renderer.create(<ErrorView />).toJSON();
    expect(tree).toMatchSnapshot();
});
