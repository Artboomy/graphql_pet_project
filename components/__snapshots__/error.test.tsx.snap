// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders default message 1`] = `
<div
  className="MuiBox-root MuiBox-root-2"
>
  <h2
    className="MuiTypography-root MuiTypography-h2 MuiTypography-colorError MuiTypography-alignCenter"
  >
    An error occurred
  </h2>
  <h6
    className="MuiTypography-root MuiTypography-h6 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
  >
    Details unknown
  </h6>
</div>
`;

exports[`Renders passed message 1`] = `
<div
  className="MuiBox-root MuiBox-root-1"
>
  <h2
    className="MuiTypography-root MuiTypography-h2 MuiTypography-colorError MuiTypography-alignCenter"
  >
    An error occurred
  </h2>
  <h6
    className="MuiTypography-root MuiTypography-h6 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
  >
    Test error message
  </h6>
</div>
`;
